### ISSUE TITLE
Push Handbook styles into guides_style_18f, begin updating Guides

### ISSUE THREAD

**Issue by mtorres253** [18F/hub#561]

**Comment by andrewmaier** [comment]
@mbland Want to pair on this?

**Comment by mbland** [comment]
Yes, but later this week. Dockerizing https://github.com/18F/knowledge-sharing-toolkit/.

**Comment by mtorres253** [comment]
@mbland @andrewmaier Have this story scheduled for grooming on Wednesday. Let's document what we want to do in the description and add an acceptance criteria before starting work on this.

**Comment by andrewmaier** [comment]
Let's incorporate the good work done by @coreycaitlin, here: https://github.com/18F/guides/issues/31
